s2/cellunion.go

```diff
@@ -553,7 +553,7 @@ func (cu *CellUnion) encode(e *encoder) {
 	}
 }
 
-// Decode decodes the CellUnion.
+// Decode decodes the CellUnion. The resulting CellUnion is not checked for validity.
```

Comment:
I still don't know what "not checked for validity" means.  Something like:
`// Note: The returned CellUnion is not necessarily valid according to IsValid().`
?